h/test/views_test.py

```diff
@@ -12,9 +13,10 @@
 from h import views
 
 
+@patch('h.views.app_config', autospec=True)
 class TestAnnotationView(unittest.TestCase):
 
-    def test_og_document(self):
+    def test_og_document(self, app_config):
```

Comment:
If you're not using these anywhere, I'd suggest you just add a single `autouse` fixture at the bottom of the module, like so:

```python
@pytest.fixture(autouse=True)
def app_config(request):
    patcher = mock.patch('h.views.app_config', autospec=True)
    patcher.start()
    request.addfinalizer(patcher.stop)
```

That way you won't need to modify the rest of the file at all.